### ISSUE TITLE
Net can't predict 0

### ISSUE THREAD

**Issue by FLming** [RaidasGrisk/tf2-crnn#1]
I find you make blank_index equals the 62(last char), but when you decode, you ignore the 0, so the net can't predict the label "0". 
I think you should change the char_dict, place the blank label in the first, and make blank_index equals 0, it will work well.

By the way, the output of the network is:
[62 62 62 62 62 62 62 62 62 62 48 17 17 17 17 18 18 18 18 62 62 62 62 62
  62 62 62 62 62 62 62 62 62 21 21 22 62 62 62 62 62 62 62 62 62 62 62]
because the zero-padding the output of ctc_greedy_decoder will be:
 [48 17 18 21 22  0  0  0  0  0]

**Comment by raidastauras** [comment]
I see your point. The line that does final decoding is ignoring the predictions of char '0'. This is because an index of 0 means two things at that point: either '0' char or zero-pad to a vector if indices.

`[decode_to_text(char_dict, [j for j in i if j != 0]) for i in decoded]`

Also, by default in tf.nn.ctc_loss the blank_label = 0. It was a bad move to change this. Thanks for pointing this out! 

Thinking of another way to change this, so that the pre-trained model is still relevant. Placing blank_index at first in char_dict, will mess up the model because all the indices will shift to the right.

**Comment by raidastauras** [comment]
Fixed this as follows:

`[[char for char in np.trim_zeros(word, 'b')] for word in decoded]`

Say the model predicts: `[15, 0, 47, 0, 0, 0]`
The above list comprehension returns: `[15, 0, 47]`

This is a sub-optimal solution, still it works. @FLming thanks for spotting this!
